macro-fullcalendar-webjar/pom.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+ *
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ *
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.xwiki.contrib</groupId>
+    <artifactId>macro-fullcalendar-pom</artifactId>
+    <version>2.2-SNAPSHOT</version>
+  </parent>
+  <artifactId>macro-fullcalendar-webjar</artifactId>
+  <name>Full Calendar Macro - WebJar</name>
+  <packaging>jar</packaging>
+  <description>Client side resources required by the fullcalendar.io integration in XWiki.</description>
+  <properties>
+    <momentjs.version>2.9.0</momentjs.version>
```

Comment:
Flamingo skin depends on momentjs since 9.5RC1 ([XWIKI-10122](https://jira.xwiki.org/browse/XWIKI-10122)) with version 2.18.1+ . It's good that you specified a lower version here, as otherwise you would have not been able to install FullCalendar since momentjs is a core extension since XWiki 9.5RC1.

Reply:
Might be better to use 2.18.1 in fullcalendar extension (i.e. the most recent possible version which does not collide with a core extension) to make sure it works with it (since that's the version you are going to get in practice in recent versions of XWiki).

Reply:
For fullcalendar the minimum support version of MomentJS was raised to v2.9.0